data/releases.yml

```diff
@@ -1,4 +1,4 @@
-- name: "1.0"
+- name: 1.0
```

Comment:
Don't change this please, it'll break things.

Reply:
Sorry I was trying to fix the issue.
The changes have been reverted.